tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -2487,6 +2494,10 @@ AllocateCudnnConvolutionBackwardFilterWorkspace(
   return scratch_allocator->AllocateBytes(stream, size_in_bytes);
 }
 
+inline bool TensorOpMathAvailable(int cc_major) {
```

Comment:
I'd change `inline` to `static` since this function is only used in this .cc file.

Reply:
Done